cr/state/committee.go

```diff
@@ -355,9 +352,6 @@ func (c *Committee) processImpeachment(height uint32, member []byte,
 
 func (c *Committee) processCRCAppropriation(tx *types.Transaction, height uint32,
 	history *utils.History) {
-	if height <= c.recordBalanceHeight {
```

Comment:
may not set NeedAppropriation to false, should change.

Reply:
fixed